Tests/UnitTests/SubscriberAttributes/PurchasesSubscriberAttributesTests.swift

```diff
@@ -409,16 +409,6 @@ class PurchasesSubscriberAttributesTests: TestCase {
             .to(equal((nil, purchases.appUserID)))
     }
 
-    func testSetAndClearOnesignalUserID() {
```

Comment:
We need to bring the tests back. You can mark the tests as `@(*, deprecated)` as well so it compiles, and they'll still run.